include/grpc/impl/codegen/grpc_types.h

```diff
@@ -216,7 +216,7 @@ typedef struct {
 #define GRPC_ARG_HTTP2_WRITE_BUFFER_SIZE "grpc.http2.write_buffer_size"
 /** Should we allow receipt of true-binary data on http2 connections?
     Defaults to on (1) */
-#define GRPC_ARG_HTTP2_ENABLE_TRUE_BINARY "grpc.http2.true_binary"
```

Comment:
We're dropping binary metadata support?

Reply:
Rather, we are muxing the two into the same bit. HFAST will represent all H2 spec breaking optimizations rather than toggle each one with its own option (until a valid use case comes along which needs one and not the other)